src/components/Sidebar/Sidebar.tsx

```diff
@@ -1,130 +1,169 @@
 import { useState, useEffect, ChangeEvent } from "react"
+import { nanoid } from 'nanoid'
 import Card from "../Card/Card"
 import './Sidebar.scss'
 import { getPlantList, searchPlants } from "../../apiCalls"
+import decorData from '../../decor_data.json'
 
 interface PlantData {
-  id: number
-  type: string
-  attributes: PlantAttributes
+    id: number
+    type: string
+    attributes: PlantAttributes
+}
+
+interface DecorData {
+    id: null;
+    type: string;
+    attributes: PlantAttributes;
 }
 
 export interface PlantAttributes {
-  hardiness: {
-    min: string
-    max: string
-  }
-  image: string
-  plant_name: string
-  plant_id: number
-  sunlight: string[]
-  type: string
+    hardiness?: {
+        min: string
+        max: string
+    }
+    image: string
+    plant_name: string
+    plant_id?: number
+    sunlight?: string[]
+    type: string
+}
+
+export interface DecorAttributes {
+    image: string
+    plant_name: string
+    type: string
 }
```

Comment:
NB: I think there is a way to `extends` types and interfaces to dry this up some

Reply:
I will look into this after I work on the key and get my videos done - if you are okay with it, let's merge this to main and take care of this later